tools/test.py

```diff
@@ -251,7 +259,10 @@ def HasRun(self, output):
     self._done += 1
     command = basename(output.command[-1])
     if output.UnexpectedOutput():
-      logger.info('not ok %i - %s' % (self._done, command))
+      status_line = 'not ok %i - %s' % (self._done, command)
+      if FLAKY in output.test.outcomes and self.flaky_tests_mode == "dontcare":
+        status_line = status_line + " # TODO : Fix flaky test"
```

Comment:
Tiny style nit: there's a mild preference for single quotes in the test runner code.